### ISSUE TITLE
Port option does not work

### ISSUE THREAD

**Issue by JayFoxRox** [danvk/RangeHTTPServer#20]
Looks like the port is ignored:

```
$ python --version
Python 3.7.0
$ python -m RangeHTTPServer 80
Serving HTTP on 0.0.0.0 port 8000 (http://0.0.0.0:8000/) ...
```

Installed using `$ pip install --user rangehttpserver`

**Comment by brotich** [comment]
pulling the project directly from Github and the `port` option is supported. seems the version on pypi is not upto date

**Comment by sakurai-youhei** [comment]
FYI: `pip install --user https://github.com/danvk/RangeHTTPServer/archive/master.zip --upgrade`

**Comment by danvk** [comment]
Anyone interested in doing an update? I'm happy to add a collaborator on PyPI.

**Comment by NicoWeio** [comment]
I'd be happy to help. :)

**Comment by DawnEve** [comment]
@danvk Hi, bossman, current version installed by pip3 still not support port, so I have to manually modify the `site-packages/RangeHTTPServer/__main__.py` file to the latest version on github.

**Comment by danvk** [comment]
Fixed by #32 and the 1.3.0 release.
